@ArnauCForgeFlow Do you know why this seemed to fix the tests on the PR but does not fix the tests during the merge?

```
2025-09-04 07:52:16,624 294 INFO odoo odoo.addons.web_systray_button_init_action.tests.test_web_systray_button_init_action.TestUI.test_ui.browser: console.groupEnd 
2025-09-04 07:52:16,625 294 ERROR odoo odoo.addons.web_systray_button_init_action.tests.test_web_systray_button_init_action.TestUI.test_ui.browser: FAILED: [3/5] Tour web_systray_button_init_action_set_tour → Step .o_app[data-menu-xmlid='base.menu_administration'].
Element (.o_app[data-menu-xmlid='base.menu_administration']) has not been found.
TIMEOUT step failed to complete within 10000 ms. 
2025-09-04 07:52:16,625 294 INFO odoo odoo.addons.web_systray_button_init_action.tests.test_web_systray_button_init_action.TestUI.test_ui: Asking for screenshot 
2025-09-04 07:52:16,627 294 INFO odoo odoo.addons.web_systray_button_init_action.tests.test_web_systray_button_init_action.TestUI.test_ui.browser: Error received after termination: {
  "trigger": ".init_action_div:has(button[name='init_action'])"
},
{
  "isActive": [
    "auto",
    "community",
    "desktop"
  ],
  "trigger": ".o_navbar_apps_menu button:enabled",
  "run": "click",
  "tooltipPosition": "bottom"
},
---------- FAILED: [3/5] Tour web_systray_button_init_action_set_tour → Step .o_app[data-menu-xmlid='base.menu_administration'] ----------
{
  "trigger": ".o_app[data-menu-xmlid='base.menu_administration']",
  "run": "click"
},
------------------------------------------------------------------------------------------------------------------------------------------
{
  "trigger": "button[name='init_action']",
  "run": "click"
},
{
  "trigger": ".o_last_breadcrumb_item:has(span:contains('Apps'))"
}, 
2025-09-04 07:52:16,627 294 INFO odoo odoo.addons.web_systray_button_init_action.tests.test_web_systray_button_init_action.TestUI.test_ui: Asking for screenshot 
2025-09-04 07:52:16,836 294 INFO odoo odoo.addons.web_systray_button_init_action.tests.test_web_systray_button_init_action.TestUI.test_ui: Screenshot in: /tmp/odoo_tests/odoo/screenshots/sc_20250904_075216_835474_TestUI.png 
2025-09-04 07:52:16,965 294 INFO odoo odoo.addons.web_systray_button_init_action.tests.test_web_systray_button_init_action.TestUI.test_ui: Closing chrome headless with pid 1161 
2025-09-04 07:52:16,966 294 INFO odoo odoo.addons.web_systray_button_init_action.tests.test_web_systray_button_init_action.TestUI.test_ui: Screenshot in: /tmp/odoo_tests/odoo/screenshots/sc_20250904_075216_965277_TestUI.png 
2025-09-04 07:52:16,966 294 INFO odoo odoo.addons.web_systray_button_init_action.tests.test_web_systray_button_init_action.TestUI.test_ui: Closing websocket connection 
2025-09-04 07:52:16,967 294 INFO odoo odoo.addons.web_systray_button_init_action.tests.test_web_systray_button_init_action.TestUI.test_ui: Terminating chrome headless with pid 1161 
2025-09-04 07:52:16,999 294 INFO odoo odoo.addons.web_systray_button_init_action.tests.test_web_systray_button_init_action.TestUI.test_ui: Removing chrome user profile "/tmp/tmpt6xnx51x_chrome_odoo" 
2025-09-04 07:52:17,018 294 INFO odoo odoo.addons.web_systray_button_init_action.tests.test_web_systray_button_init_action.TestUI.test_ui: waiting for threads: [] 
2025-09-04 07:52:17,019 294 INFO odoo odoo.addons.web_systray_button_init_action.tests.test_web_systray_button_init_action: ====================================================================== 
2025-09-04 07:52:17,019 294 ERROR odoo odoo.addons.web_systray_button_init_action.tests.test_web_systray_button_init_action: FAIL: TestUI.test_ui
Traceback (most recent call last):
  File "/__w/web/web/web_systray_button_init_action/tests/test_web_systray_button_init_action.py", line 20, in test_ui
    self.start_tour(
  File "/opt/odoo/odoo/tests/common.py", line 2266, in start_tour
    return self.browser_js(url_path=url_path, code=code, ready=ready, timeout=timeout, success_signal="tour succeeded", **kwargs)
  File "/opt/odoo/odoo/tests/common.py", line 2240, in browser_js
    self.fail('%s\n\n%s' % (message, error))
AssertionError: The test code "odoo.startTour('web_systray_button_init_action_set_tour', {"stepDelay": 100, "keepWatchBrowser": false, "debug": false, "startUrl": "/web", "delayToCheckUndeterminisms": 0})" failed

FAILED: [3/5] Tour web_systray_button_init_action_set_tour → Step .o_app[data-menu-xmlid='base.menu_administration'].
Element (.o_app[data-menu-xmlid='base.menu_administration']) has not been found.
TIMEOUT step failed to complete within 10000 ms.
```

--- a/web_company_color/models/res_company.py
+++ b/web_company_color/models/res_company.py
@@ -115,7 +115,7 @@ class ResCompany(models.Model):
             background-color: %(color_navbar_bg_hover)s !important;
           }
         }
-        .o_menu_systray .o-dropdown .dropdown-toggle {
+        .o_menu_systray button{
             color: %(color_navbar_text)s !important;
             &:hover, &:focus, &:active, &:focus:active {
                 background-color: %(color_navbar_bg_hover)s !important;

